easybuild/easyconfigs/p/PSI4/PSI4-1.3.2-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'EB_PSI'
+
+name = 'PSI4'
+version = '1.3.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.psicode.org/'
+description = """PSI4 is an open-source suite of ab initio quantum chemistry programs designed for
+ efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform
+ computations with more than 2500 basis functions running serially or in parallel."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'usempi': True}
+
+github_account = 'psi4'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['PSI4-%(version)s_fix_cmake_release.patch']
+checksums = [
+    'ed76c67803b6420f35f57a6dd31c47108b9145b8c9fced5c94cdc179f6b5fbf3',  # v1.3.2.tar.gz
+    'c399b6a90e587976f7ec98a3f05c3b62525d015639f8352004b5bd4bea696eda',  # PSI4-1.3.2_fix_cmake_release.patch
+]
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    # Perl needed for the ctest test suite during the installation, not for the pytest test suite after the installation
+    # ('Perl', '5.30.0'),
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    #  PCMSolver recent versions fail with recent intel compilers.
+    #  Tried with intel/201{7,8}b, and versions 1.1.12, and 1.2.1
+    #  See in more details: https://github.com/PCMSolver/pcmsolver/issues/159
+    ('PCMSolver', '1.2.3', versionsuffix),
+    ('CheMPS2', '1.8.9'),
+    ('pytest', '6.0.1', versionsuffix),
+    ('networkx', '2.4', versionsuffix),
+    ('deepdiff', '5.0.2', versionsuffix),
+    ('libxc', '4.3.4'),
+    ('pybind11', '2.4.3', versionsuffix),
+    ('pydantic', '1.6.1', versionsuffix),
+    ('Pint', '0.14', versionsuffix),
+]
+
+configopts = '-DENABLE_MPI=ON -DENABLE_PLUGINS=ON -DENABLE_XHOST=OFF '
+#  Install python mudule to the standard location instead of lib
+configopts += '-DPYMOD_INSTALL_LIBDIR=/python%(pyshortver)s/site-packages '
+#  Add bundled (downloaded) packages
+# -DENABLE_simint=ON does not work with intel/2018, so have to make with GCCcore
```

Comment:
Does this comment still apply?

Reply:
They still have a check for Intel < 2018 in the CmakeLists.txt for simint so it does apply as such, and since I have no idea on how to test this I'd rather keep it out.